### PR TITLE
doc: Update the volume size doc

### PR DESCRIPTION
**Pull request by shuo-wu**
Longhorn/longhorn#4158

### CODE REVIEW

File: content/docs/1.3.0/volumes-and-nodes/volume-size.md [hunk 1]
```diff
@@ -91,24 +105,37 @@ Here we summarize the important things related to disk space usage we have in th
...
         ```
-        (N + 1 + 1) x head/snapshot average actual size
+        (M + N + 1 + 1 + 1 + 1) x head/snapshot average actual size
         ```
 
-        where N is the number of snapshots retained, the 1st 1 is for the system snapshot, and the 2nd 1 is for the temporary space that may be required by snapshot deletion.
+        where `M` is the snapshots created by users manually, `N` is the snapshot recurring job retain number, the 1st `1` means the volume head, the 2nd `1` is the system snapshots created by rebuilding or expansion operations, the 3rd `1` is the extra snapshot before the recurring job starts cleanup, and the 4th `1` is for the temporary space that may be required by snapshot deletion.
+   - If users don't want snapshot at all. The formula would be
+       ```
+       (1 + 1 + 1) x head/snapshot average actual size
```
Comment (PhanLe1010) [inline content/docs/1.3.0/volumes-and-nodes/volume-size.md:127]: typo: missing one number 1

File: content/docs/1.3.0/volumes-and-nodes/volume-size.md [hunk 2]
```diff
@@ -91,24 +105,37 @@ Here we summarize the important things related to disk space usage we have in th
...
+       ```
+
+       where the 1st `1` means the volume head, the 2nd `1` is the first system snapshot created by rebuilding or expansion operations, the 3rd `1` is the second system snapshot, and the 4th `1` is for the temporary space that may be required by snapshot deletion/merging the 1st system snapshot with the 2nd one.
```
Comment (PhanLe1010) [inline content/docs/1.3.0/volumes-and-nodes/volume-size.md:130]: This fomular is bigger than the number that we gave UiPath (3x). Should we mention that this is the absolute worst case? The arrange case should be 3x?

Comment (shuo-wu) [reply content/docs/1.3.0/volumes-and-nodes/volume-size.md:130]: Not sure. Typically 3x is large enough. This is actually the corner case:
1. Volume head is full
2. 1st Rebuilding is triggered. (The purge of this rebuilding cannot do anything.)
3. Before there is another pruning coming, the volume head suddenly becomes full and the 2nd rebuilding is triggered.
4. During the purge (for the 2nd rebuilding), the volume head becomes full

File: content/docs/1.3.0/volumes-and-nodes/volume-size.md (hunk 1 above)
Comment (shuo-wu) [reply content/docs/1.3.0/volumes-and-nodes/volume-size.md:127]: Oh YES. Thank you for pointing that!

File: content/docs/1.3.0/volumes-and-nodes/volume-size.md (hunk 2 above)
Comment (derekbit) [reply content/docs/1.3.0/volumes-and-nodes/volume-size.md:130]: One question: will the snapshot purging and rebuilding/expanding happen simultaneously?

Comment (shuo-wu) [reply content/docs/1.3.0/volumes-and-nodes/volume-size.md]: IIRC, No. The snapshot purge is the last step of the rebuilding, which happens after the new replica becomes RW.

**Review by innobead — COMMENTED** [review]
LGTM. After discussing with @shuo-wu, just one part related to the no-user snapshot case, and we probably need to explain a bit why there will be 2 system snapshots following the volume head.

**Review by innobead — APPROVED** [review]
LGTM

**Comment by innobead** [comment]
cc @longhorn/qa

**Pull request merged**